datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/security/DocusafeAppInitializer.java

```diff
@@ -0,0 +1,15 @@
+package de.adorsys.datasafe.rest.impl.security;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.filter.DelegatingFilterProxy;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+
+public class DocusafeAppInitializer implements WebApplicationInitializer {
+    @Override
+    public void onStartup(ServletContext servletContext) throws ServletException {
```

Comment:
Is it necessary?
You add this in WebSecurityConfigurerAdapter. Most probably this class is not needed